### DISCUSSION TITLE
Problem with sphinx 4.0.2 with Python 3.10

### DISCUSSION THREAD

**Discussion by mmdbalkhi** [flaskcwg/translation-coordination#10] [Q&A] [answered]
# Problem with sphinx 4.0.2 with Python 3.10

I installed Python 3.10 today and it seems that the version of the sphinx that was in the project has a problem with this version of Python. I updated sphinx and the problem was fixed.

Should I write new versions (4.3.1) of the sphinx in the requirements files? If yes how do I do this automatically?

## Useful links

- <https://github.com/sphinx-doc/sphinx/issues/9562>

**Comment by greyli** [comment, marked as answer]
Flask will update all dependencies soon, you can wait then sync changes from the upstream repo.

**Reply by mmdbalkhi** [reply]
Hello @greyli . If you saw this [log](https://github.com/mmdbalkhi/flask-docs-fa/runs/4667766433?check_suite_focus=true), this problem exists in this older version of the sphinx. What do you think should be done?
